app/app.js

```diff
@@ -1,9 +1,10 @@
-import Ember from 'ember';
-import Resolver from './resolver';
+import Application from '@ember/application';
 import loadInitializers from 'ember-load-initializers';
+
+import Resolver from './resolver';
```

Comment:
can you put resolver before loadInitializers? or did the order change in blueprints?